pathod/utils.py

```diff
@@ -59,7 +59,8 @@ def inner_repr(s):
         Returns the inner portion of a string or unicode repr (i.e. without the
         quotes)
     """
-    if six.PY2 and isinstance(s, unicode):
+    if (six.PY2 and isinstance(s, unicode)) or \
+            (six.PY3 and isinstance(s, bytes)):
```

Comment:
using `six.PY3` is usually not a good idea, because this might break if Python 4 gets released. We should assume future-compatibility and only check for legacy-incompatibilities.